docs/Documentation/publish-via-cli.md

```diff
@@ -0,0 +1,27 @@
+# Publish from Command Line 
+
+![Publish Via CLI](../../assets/images/publish-via-cli.png)
+
+1. Install the @stoplight/cli package from NPM 
+
+```
+Npm i -g @stoplight/cli
+
+# The “stoplight” cli should now be available globally 
+
+Stoplight help 
+```
+
+2. Make sure you are in the root directory of the local git repository associated with this project 
+
+```
+cd directory/to/analyze 
+```
+
+3. Run the analyze command, passing in the secret token associated with this project, and the Stoplight API host. This will analyze all the design-related files in the local project directory, and send the information to Stoplight to be displayed as published docs, included in the search explorer, etc. 
```

Comment:
Remove `and the Stoplight API host`.

Maybe indicate that the project token is available in the project settings publish section?